app/models/miq_alert.rb

```diff
@@ -1,6 +1,13 @@
 class MiqAlert < ApplicationRecord
   include UuidMixin
 
+  AVAILABLE_SEVERITIES = {
```

Comment:
What does the word AVAILABLE signify here and in the method below?  Is it possible to have*more* severities?  Might be better to just remove the word entirely